src/commands/initProjectForVSCode/InitVSCodeStep/InitVSCodeStepBase.ts

```diff
@@ -133,7 +133,7 @@ export abstract class InitVSCodeStepBase extends AzureWizardExecuteStep<IProject
         const matchingTaskLabels: string[] = [];
         for (const existingTask of existingTasks) {
             const existingLabel = this.getTaskLabel(existingTask);
-            if (existingLabel && newTasks.some(newTask => existingLabel === this.getTaskLabel(newTask))) {
+            if (existingLabel && newTasks.some(newTask => existingLabel === this.getTaskLabel(newTask) && existingTask.type !== 'func')) {
```

Comment:
This doesn't do quite exactly what I was expecting.

This [code](https://github.com/microsoft/vscode-azurefunctions/blob/bf447689cfe91df031e97ce2e1261106f59dd290/src/commands/initProjectForVSCode/InitVSCodeStep/InitVSCodeStepBase.ts#L152-L162) will make any task with `type` `func` return `func: host start`. 

We want to respect the label property (if it exists).  Then we should compare it to `export const hostStartTaskName: string = `${func}: ${hostStartCommand}`;` that lives in the `constants.ts` file. If it matches, then we should push it into `matchingTaskLabels`

Also, we don't need to check the `type` on the existingTask. There are times that we will want to replace `func` tasks. Like, if somebody changed their `func: start host` command and are trying to re-initialize their project to get it back to how it was.